internal/download/manager.go

```diff
@@ -4,18 +4,17 @@ import (
 	"io/ioutil"
 	"path/filepath"
 
+	"github.com/ActiveState/cli/internal/failures"
 	"github.com/ActiveState/cli/internal/fileutils"
 	"github.com/ActiveState/cli/internal/locale"
-
-	"github.com/ActiveState/cli/internal/failures"
 	"github.com/ActiveState/cli/internal/logging"
 	"github.com/ActiveState/cli/internal/progress"
 )
 
 // Manager is our main download manager, it takes care of processing the downloads and communicating progress
 type Manager struct {
 	WorkerCount int
-	failure     *failures.Failure
+	err         error
```

Comment:
This was out of scope for your story. Since the work is done let's leave it in, but please scope yourself down next time.

Reply:
It seemed to make sense at the time since I would have had to, otherwise, added failure wrapping to the errors returned by the download func.

Reply:
That would have been much simpler than refactoring all of this.